.gitbook/1-users/3-data-exchange/1-consume-data.md

```diff
@@ -0,0 +1,93 @@
+# Consume data
+
+Data consumers can open data deals by specifying the type, the quantity, and the pricing of the data that they are willing to consume. 
+
+Data deals are registered in the Panacea public blockchain, so all data providers can find data deals, which match their data.
+
+
+## Create a data deal
+
+Broadcast the following `create-deal` transaction with specifying data schema, a data count, and a budget in the deal-file in JSON format.
+```bash
+panacead tx datadeal create-deal ${deal-file-path} \
+  --from ${data-consumer-addr} \
+  --chain-id ${chain-id} \
+  --gas auto --gas-adjustment 1.30 --gas-prices 5umed \
+  --node ${chain-node-rpc-addr}
+```
+
+For `deal-file-path`, create a following JSON file.
+```json
+{
+  
+}
```

Comment:
This JSON example(?) is empty.
If it is not intentional, here is an example:
```json
{
  "data_schema": [
    "http://json.schemastore.org/github-issue-forms.json"
  ],
  "budget": {
    "denom": "umed",
    "amount": "1000000"
  },
  "max_num_data": 10,
  "consumer_address": "panacea1..."
}

Reply:
Yeah. I was hesitating since I don't wanna use the data schema `github-issue-forms` which is not valuable for data exchange :) I'll make a sample data schema for "not-that-sensitive" medical data.

Reply:
done 9dd634eb3a4f07a90fb2eb9a4b8fb2ffb54a2183